File: leetcode/Easy/removeElement.js
var removeElement = function(nums, val) {
    let ans = []
    let k = 0
    for (let j = 0; j < nums.length; j++) {
      if (nums[j] !== val) {
          ans.push(nums[j])
          k++
        } 
    }
    for (let i = 0; i<ans.length; i++) {
        nums[i] = ans[i]
    }
    return k;
};

File: leetcode/Easy/removeElement.py
# Runtime - 48.59% and Memory - 33.73%

class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i = 0
        k = 0
        for j in range(len(nums)):
            if nums[j] != val:
                nums[i] = nums[j]
                i+=1
                k+=1
        return k